### DISCUSSION TITLE
Allow setting "Text Value" for Autocomplete input

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#6879] [Feature Requests]
At the moment I can setup this:

<img width="665" alt="Screen Shot 2021-07-19 at 16 42 41" src="https://user-images.githubusercontent.com/4686040/126169362-b80d6a5f-7cb2-4561-ab62-c1658a76b4c4.png">

I was wondering if we can setup it same we do for dropdown, `Text`, `Value` smth like this:

```js
const result = {
    predictions: [
      {
        structured_main_text: 'My Text',
        value: '1'
      }
    ]
  }
```

Thx.

**Comment by rijkvanzanten** [comment]
Not currently possible, but I like the idea!

**Reply by botzill** [reply]
Yes, and we can have as well possibility to select multiple items.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
